spec/javascripts/packages/device/index-spec.js

```diff
@@ -1,26 +1,93 @@
-import { isLikelyMobile, hasMediaAccess, isCameraCapableMobile } from '@18f/identity-device';
+import {
+  isLikelyMobile,
+  hasMediaAccess,
+  isCameraCapableMobile,
+  isIPad,
+} from '@18f/identity-device';
+
+describe('isIPad', () => {
+  let originalUserAgent;
+  beforeEach(() => {
+    originalUserAgent = navigator.userAgent;
+    navigator.maxTouchPoints = 0;
+    Object.defineProperty(navigator, 'userAgent', {
```

Comment:
This spec predates the addition of the [`useDefineProperty` test helper](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/test-helpers/use-define-property.ts), which can help deal with some of the boilerplate around defining and restoring properties this way.

```ts
import { useDefineProperty } from '@18f/identity-test-helpers';

describe('isIPad', () => {
  const defineProperty = useDefineProperty();

  context('with ipad (old user agent format)', () => {
    beforeEach(() => {
      defineProperty(navigator, 'userAgent', {
        value:
          'Mozilla/5.0(iPad; U; CPU iPhone OS 3_2 like Mac OS X; en-us) AppleWebKit/531.21.10 (KHTML, like Gecko) Version/4.0.4 Mobile/7B314 Safari/531.21.10',
      });
    });

    it('returns true', () => {
      expect(isIPad()).to.be.true();
    });
  });
});
```